data-source/src/main/scala/tech/ytsaurus/spyt/serializers/YtLogicalType.scala

```diff
@@ -84,60 +119,591 @@ sealed abstract class CompositeYtLogicalTypeAlias(name: String,
 }
 
 object YtLogicalType {
+
   import tech.ytsaurus.spyt.types.YTsaurusTypes.instance.sparkTypeFor
 
-  case object Null extends AtomicYtLogicalType("null", 0x02, ColumnValueType.NULL, TiType.nullType(), NullType)
+  case object Null extends AtomicYtLogicalType("null", 0x02, ColumnValueType.NULL, TiType.nullType(), NullType) {
+    override def ytGettersFromList(ytGetter: InternalRowYTGetters): ytGetter.FromList = new ytGetter.FromListToNull {
```

Comment:
Can we have an abstract implementations of YtGetters so we don't need to implement it in every case object? Consider adding extra parameters to AtomicYtLogicalType